scipy/differentiate/_differentiate.py

```diff
@@ -755,23 +762,26 @@ def jacobian(f, x, *, tolerances=None, maxiter=10,
             - ``-1`` : The error estimate increased, so iteration was terminated.
             - ``-2`` : The maximum number of iterations was reached.
             - ``-3`` : A non-finite value was encountered.
-            - ``-4`` : Iteration was terminated by `callback`.
 
         df : float array
             The Jacobian of `f` at `x`, if the algorithm terminated
             successfully.
         error : float array
             An estimate of the error: the magnitude of the difference between
-            the current estimate of the derivative and the estimate in the
-            previous iteration.
+            the current estimate and the estimate in the previous iteration.
```

Comment:
Changed inadvertently during merge conflict resolution.
```suggestion
            the current estimate of the derivative and the estimate in the
            previous iteration.
```